style.css

```diff
@@ -0,0 +1,612 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&family=Poppins:wght@800&display=swap');
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.navbar-one {
+  display: none;
+  justify-content: flex-end;
+  background-color: #1c1f23;
+  gap: 20px;
+  padding: 10px;
+}
+
+.nav-item {
+  color: #fff;
+  text-decoration: none;
+  transition: 1s;
+}
+
+.nav-item:hover {
+  transform: scale(1.2);
+  color: #ec6e42;
+}
+
+.navbar-two {
+  background-color: #eee;
+}
+
+.humburger {
+  width: 90px;
+  height: 70px;
+  padding: 10px;
+}
+
+.icon-bar {
+  display: block;
+  margin: 6px auto;
+  height: 8px;
+  width: 50px;
+  color: #272a31;
+  background-color: #272a31;
```

Comment:
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design 🔥 Well done!